README.md

```diff
@@ -0,0 +1,86 @@
+# TransformerQAExecutor
+
+**TransformerQAExecutor** wraps a question-answering model from huggingface. Given some questions and paragraphs/contexts, it extracts the relevant answers from the given paragraphs/contexts.
+
+**TransformerQAExexecutor** receives [`Document`s](https://docs.jina.ai/fundamentals/document/) where each `Document`'s `text` is the question, and its `matches`'s texts are the contexts/paragraphs where answers will be extracted from.
+
+## Usage
+
+Use the prebuilt images from Jina Hub in your Flow:
+
+```python
+from jina import Flow, Document
+
+f = Flow().add(uses='jinahub+docker://TransformerQAExecutor')
+
+doc = Document(
+    content='what is the color of an apple?',
+    matches=[
+        Document(text='Apple is a pome fruit that is red in color.'),
+        Document(text='Banana is yellow.')
+    ]
+)
+
+with f:
+    doc = f.search(inputs=doc, return_results=True).docs[0]
+```
+
+After search, to get let's say the top 1 answer, paragraph where top 1 answer is extracted from and the confidence score of the top 1 answer:
```

Comment:
This sentence is too long and confusing. Clarify and split it up.

Reply:
Addressed.